decidim/decidim
# Fix order when filtering Meetings

#### :tophat: What? Why?

After the introduction of checkboxes on Meeting date filters we've lost the possibility of having them sorted by start date. This PR fixes it. Upcoming and past meetings are ordered by the start date, where the most recent meeting is shown first. For this to make sense, I've changed the checkboxes to radio buttons. 

I've also fixed a bug on seeds where we will have all the meetings on a Participatory Space with the same date, making it difficult to check this. 

#### :pushpin: Related Issues
 
- Fix #8589 
- Fix #9113

#### Testing
 
1. Run seeds on a clean database, go to a meeting filter page on a process. See that the meetings aren't anymore all on the same day. 
2. See that in this page the order changes when filtering by "Date":
   1. Upcoming meetings are shown first the most recent ones and then go to the future
   2. Past meetings are shown first the most recent ones and then go to the past
   3. "All" are shown from past to future

### :camera: Screenshots

#### Before

![image](https://user-images.githubusercontent.com/717367/176915745-61d5ede9-346b-4d09-b6cf-d2fb5f6176f5.png)

#### After
![Selection_256](https://user-images.githubusercontent.com/717367/176915764-23c4b57c-a755-4284-8dba-cdac05d20496.png)


:hearts: Thank you!
